Xwt.XamMac/Xwt.Mac/ContextBackendHandler.cs

```diff
@@ -309,12 +307,14 @@ public override void DrawImage (object backend, ImageDescription img, Rectangle
 			ctx.TranslateCTM ((float)(destRect.X - (srcRect.X * rx)), (float)(destRect.Y - (srcRect.Y * ry)));
 			ctx.ScaleCTM ((float)rx, (float)ry);
 
+			NSImage image = img.ToNSImage (copyNSImage: false);
```

Comment:
Since `copyNSImage: false` means returning the NSImage backend as is, it would be more simple to just do this operation here. That is, instead of creating a new overload of ToNSImage, just do `image = (NSImage)img.Backend` here.